src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParentheses15Extensions.java

```diff
@@ -22,7 +22,7 @@
 @MyAnnotation1(name = "ABC", version = 1)
 public class InputUnnecessaryParentheses15Extensions
 {
-
+    boolean flag = ("abc" == "done"); // violation 'parentheses around assignment right-hand side'
```

Comment:
We can extend test input in this case, but please create a new, similar class in this file instead of modifying this one.

Reply:
Done.